server/src/controllers/score_controller.rs

```diff
@@ -0,0 +1,89 @@
+use actix_web::{web, http, HttpResponse, HttpRequest, guard};
+use crate::models::score_model::Score;
+
+pub fn routes(cfg: &mut web::ServiceConfig) {
+    cfg.service(web::scope("/scores")
+        .route("", web::get().to(get_all))
+        .route("", web::post()
+            .guard(guard::Header("content-type", "application/json"))
+            .to(create))
+        .route("/{id}", web::put()
+            .guard(guard::Header("content-type", "application/json"))
+            .to(edit))
+        .route("/{id}", web::get().to(get_one))
+        .route("/{id}", web::delete().to(delete))
+    );
+}
+
+fn get_all(_req: HttpRequest) -> HttpResponse {
+    HttpResponse::Ok().json(Score::get_all())
+}
+
+fn get_one(req: HttpRequest) -> HttpResponse  {
```

Comment:
I think for this controller it would be better to add to `get_all`  filters to filter by game and/or by player. 
My reasoning is that we will pbly never get all the scores in the db, but we will definitely want to get all the scores for a game and a player.